packages/osquery_elastic_managed/changelog.yml

```
@@ -1,4 +1,10 @@
 # newer versions go on top
+- version: "0.1.4"
+  changes:
+    - description: Updated datastream naming convention for "osquery elastic managed" intergration to logs-osquery_managed.result-default.
+      type: breaking change
```

So it passes validation.

```suggestion
      type: breaking-change
```